package-overrides/github/angular/bower-material@0.8.2.json

```diff
@@ -0,0 +1,18 @@
+{
+  "main": "angular-material",
+  "shim": {
+    "exports": "'ngMaterial'",
```

Comment:
Yes the module name needs to be the first property of shim, with the deps and exports object as its value.